src/App.js

```diff
@@ -25,45 +25,37 @@ const App = ({ services, zipConversionEndpoint }) => {
 
 	useAppInitializer(ctsapiclient, zipConversionEndpoint);
 
+	const {
+		BasicSearchPagePath,
+		ResultsPagePath,
+		TrialDescriptionPagePath,
+		AdvancedSearchPagePath,
+	} = useAppPaths();
+
 	return (
 		<>
 			{!appHasBeenInitialized ? (
 				// Is initializing, show loading screen.
 				<></>
 			) : initErrorsList.length === 0 ? (
 				<>
```

Comment:
you no longer need the fragment since the PrintContextProvider is now a single node